dev/integration_tests/flutter_gallery/perf.json

```diff
@@ -0,0 +1 @@
+{"targets":[{"name":"gen_localizations","skipped":true,"succeeded":true,"elapsedMilliseconds":1},{"name":"gen_dart_plugin_registrant","skipped":true,"succeeded":true,"elapsedMilliseconds":4},{"name":"kernel_snapshot","skipped":true,"succeeded":true,"elapsedMilliseconds":459},{"name":"android_aot","skipped":true,"succeeded":true,"elapsedMilliseconds":45},{"name":"aot_android_asset_bundle","skipped":true,"succeeded":true,"elapsedMilliseconds":545},{"name":"android_aot","skipped":true,"succeeded":true,"elapsedMilliseconds":41},{"name":"android_aot","skipped":true,"succeeded":true,"elapsedMilliseconds":55},{"name":"android_aot_bundle_release_android-arm","skipped":true,"succeeded":true,"elapsedMilliseconds":45},{"name":"android_aot_bundle_release_android-arm64","skipped":true,"succeeded":true,"elapsedMilliseconds":41},{"name":"android_aot_bundle_release_android-x64","skipped":true,"succeeded":true,"elapsedMilliseconds":41},{"name":"_composite","skipped":true,"succeeded":true,"elapsedMilliseconds":0}]}
```

Comment:
What is this file and why was it added?

Reply:
It should not have been added, it was a mistaken add from a different terminal window. Reverted and re-did the commit, goal of commit was to just add the one missed gradle-wrapper file. 

Confirmed with grep that all gradle-wrapper files are now using 7.3+.